pkg/check/version.go

```diff
@@ -21,14 +21,17 @@ func versionCheck(_ context.Context, r Resources, _ loggers.Advanced) error {
 		return err
 	}
 	defer db.Close()
-	// This ensures that we first return an error like
```

Comment:
After removing this, will the end user not ever see any errors about connectivity or authentication issues? That doesn't seem very friendly.

Reply:
Good question, I think you are right. Let me fix it to restore this.